frontend/src/http.js

```diff
@@ -1,12 +1,9 @@
 import axios from 'axios';
 
-// const axiosParams = function axiosParams() {
-//   const params = new URLSearchParams();
-//   return params;
-// };
+const production = process.env.NODE_ENV === 'production';
```

Comment:
We need to set up this env vars before the `yarn build` in the Dockerfile, right?

Reply:
this env var is automatically set by `yarn build`